tgui/packages/tgui/interfaces/PreferencesMenu/MainPage.tsx

```diff
@@ -464,13 +461,10 @@ export const getRandomization = (
 
 export const MainPage = (props: { openSpecies: () => void }) => {
   const { act, data } = useBackend<PreferencesMenuData>();
-  const [currentClothingMenu, setCurrentClothingMenu] = useLocalState<
+  const [currentClothingMenu, setCurrentClothingMenu] = useState<
     string | null
-  >('currentClothingMenu', null);
```

Comment:
actual decent usage of casting usestate with a union type. too bad about null though